src/Provider/GoogleMaps/Tests/GoogleMapsTest.php

```diff
@@ -282,6 +284,33 @@ function (RequestInterface $request) use (&$uri) {
         );
     }
 
+    public function testBusinessQueryWithPrivateKeyAndChannel()
+    {
+        $uri = '';
+
+        $provider = GoogleMaps::business(
+            $this->getMockedHttpClientCallback(
+                function (RequestInterface $request) use (&$uri) {
+                    $uri = (string) $request->getUri();
+                }
+            ),
+            'foo',
+            'bogus',
+            null,
+            null,
+            'bar'
+        );
+
+        try {
+            $provider->geocodeQuery(GeocodeQuery::create('blah'));
+        } catch (InvalidServerResponse $e) {
+        }
+        $this->assertEquals(
```

Comment:
Maybe do a `asserRegex` instead. This test is likely to change when we add a new parameter.

Reply:
I just copied it from the other test. I can replace both if you want.